src/version/version.service.ts

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../prisma/prisma.service';
+import { Version } from '.prisma/client';
+
+@Injectable()
+export class VersionService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async create(version: string): Promise<Version> {
+    return this.prisma.version.create({
+      data: {
+        version,
+      },
+    });
+  }
+
+  async getLatest(): Promise<Version | null> {
+    return await this.prisma.version.findFirst({
+      orderBy: {
+        id: 'desc',
```

Comment:
I doubt that we'll have this out of order, but what do you think about sorting by the version?

Reply:
I thought about that, but landed here because it allows for the ability to rollback / change version patterns without needing to refactor.